marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/bulk-data-services/job/deleteJobs.api

```diff
@@ -0,0 +1,24 @@
+{
+    "endpoint": "/data-hub/5/bulk-data-services/job/deleteJobs.sjs",
```

Comment:
We have a data-hub/5/data-services/bulk directory already; let's toss this there as well. We may end up reorganizing those later, but since that directory exists, it's consistent for now to keep using it.